utils/src/lib.rs

```diff
@@ -884,106 +988,6 @@ pub fn abi_to_solidity(contract_abi: &Abi, mut contract_name: &str) -> Result<St
     })
 }
 
-#[derive(Debug)]
```

Comment:
I just moved up this part of the linking step so I didn't have to scroll as much